package.json

```diff
@@ -51,10 +51,12 @@
     "eslint": "^1.10.3",
     "eslint-config-kentcdodds": "^5.0.0",
     "eslint-plugin-mocha": "^1.1.0",
+    "execa": "^0.2.2",
```

Comment:
These should both be regular dependencies because they're used on install right?